superbench/common/utils/__init__.py

```diff
@@ -11,7 +11,7 @@
 from superbench.common.utils.process import run_command
 from superbench.common.utils.topo_aware import gen_topo_aware_config
 from superbench.common.utils.gen_traffic_pattern_config import (
-    gen_pair_wise_config, gen_traffic_pattern_host_group, gen_ibstat
+    gen_pair_wise_config, gen_traffic_pattern_host_groups, gen_ibstat
```

Comment:
It's OK to change the function name this time. But it would be great to keep it in future. It's very strange to change the interface time-to-time.

Reply:
sure, thanks for the suggestion.